pkg/controller/controller.go

```diff
@@ -225,12 +226,17 @@ func (p *csiProvisioner) Provision(options controller.VolumeOptions) (*v1.Persis
 		glog.V(3).Infof("create volume rep: %+v", *rep.VolumeInfo)
 	}
 
+	annotations := map[string]string{provisionerIDAnn: p.identity}
+	attributesString, err := json.Marshal(rep.VolumeInfo.Attributes)
+	if err != nil {
+		glog.V(2).Infof("fail parsing volume attributes: %+v", rep.VolumeInfo.Attributes)
+	} else {
+		annotations["csi.volume.kubernetes.io/volume-attributes"] = string(attributesString)
```

Comment:
please define a constant for the annotation name

Reply:
done